components/site-header-ru.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <div
+    class="site-header"
+    :class="{
+      'site-header--menu-showed': mobileMenuShowed
+    }"
+  >
+    <div class="site-header__inner">
+      <span class="site-header__menu-toggler" @click="toggleMobileMenu">
+        Menu
+      </span>
+
+      <a
+        class="site-header__logo"
+        href="/ru"
+      >
+        <img
+          src="~/assets/images/hawk-logo.png"
+          alt="Hawk.so logo"
+        >
+        Hawk
+      </a>
+      <div class="site-header__menu">
+        <a href="https://docs.hawk.so/">Документация</a>
+        <div
+          class="site-header__menu__local-version"
+          href="/download">
+            <img src="~/assets/svg/locale-version.svg" class="site-header__menu__local-version-svg"/>
+            <div class="text-container">Локализованная версия</div>
+        </div>
```

Comment:
lets remove the Localized version label from mobile menu 
![image](https://github.com/user-attachments/assets/831bcd94-d2d9-44c2-9cc5-0f3c3e84e33f)